.gitignore

```diff
@@ -22,3 +21,4 @@ tmp
 mkmf.log
 .idea/
 .rubocop-http*
+test/reports/*
```

Comment:
minor, but I would move this up to replace L14 instead of be at the end.. just to group with the other `test/*`